hw4_zd2.js

```diff
@@ -0,0 +1,10 @@
+function User(firstName, lastName, age) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.print = function () {
```

Comment:
This is not a correct and not optimized way of adding method.

Reply:
Done